domains/atharv.json

```diff
@@ -10,8 +10,12 @@
     "CNAME": {
        "name": "atharv", 
        "value": "portfolio-atharv-r-gachchi.vercel.app"
-     }
+     },
+    "TXT": {
+        "name": "_vercel.atharv",
```

Comment:
You might need to remove .atharv, it is created currently as _vercel.atharv.atharv.is-a-good.dev which is most likely not intended.